roles/azimuth_caas_operator/defaults/main.yml

```diff
@@ -2,7 +2,7 @@
 # The chart to use
 azimuth_caas_operator_chart_repo: https://stackhpc.github.io/azimuth-caas-operator
 azimuth_caas_operator_chart_name: azimuth-caas-operator
-azimuth_caas_operator_chart_version: 0.5.0
+azimuth_caas_operator_chart_version: '0.6.0'
```

Comment:
Nit:
```suggestion
azimuth_caas_operator_chart_version: 0.6.0
```

Reply:
Nvm, turns out this is safer and all other non-quoted instances in this file should be fixed instead...